src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java

```diff
@@ -0,0 +1,149 @@
+  @JsonProperty(value = "acquirerCode", required = true)
+  String acquirerCode;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 1)
+  @Pattern(regexp = "\\d{2}", message = "Operation type length must match [0-9]{2}")
+  @JsonProperty(value = "operationType", required = true)
+  String operationType;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 2)
+  @Pattern(regexp = "\\d{2}", message = "Circuit type length must match [0-9]{2}")
+  String circuitType;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 3)
+  @Pattern(regexp = "[a-zA-Z0-9]{64}",
+      message = "HPAN length must be 64 alphanumeric char at max")
+  String hpan;
+
+  //ISO8601
+  @NotNull
+  @CsvBindByPosition(position = 4)
+  @CsvDate(value = "yyyy-MM-dd'T'HH:mm:ss.SSS[XX][XXX]")
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+  @JsonSerialize(using = OffsetDateTimeSerializer.class)
+  OffsetDateTime trxDate;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 5)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,255}",
+      message = "ID trx acquirer length must be 255 alphanumeric char at max")
+  String idTrxAcquirer;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 6)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,255}",
+      message = "ID trx issuer length must be 255 alphanumeric char at max")
+  String idTrxIssuer;
+
+  @CsvBindByPosition(position = 7)
+  @Pattern(regexp = "[a-zA-Z0-9]{0,255}",
+      message = "Correlation ID length must be 255 alphanumeric char at max")
+  String correlationId;
+
+  @NotNull
+  @CsvBindByPosition(position = 8)
+  BigDecimal amount;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 9)
+  @Pattern(regexp = "978",
+      message = "Currency must be 978 (fixed value, ISO 4217 for Euro)")
+  String amountCurrency;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 10)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,255}",
+      message = "Acquirer ID length must be 255 alphanumeric char at max")
+  String acquirerId;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 11)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,255}",
+      message = "Merchant ID length must be 255 alphanumeric char at max")
+  String merchantId;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 12)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,255}",
+      message = "Terminal ID length must be 255 alphanumeric char at max")
+  String terminalId;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 13)
+  @Pattern(regexp = "\\d{6}|\\d{8}",
+      message = "BIN length must match [0-9]{6}|[0-9]{8}")
+  String bin;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 14)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,5}", message = "MCC length must be 5 alphanumeric char at max")
+  String mcc;
+
+  @CsvBindByPosition(position = 15)
+  @Pattern(regexp = "[a-zA-Z0-9]{11}|[a-zA-Z0-9]{16}",
+      message = "Fiscal code must be 11 or 16 alphanumeric char")
+  String fiscalCode;
```

Comment:
is fiscalCode not mandatory?

Reply:
I enhanced fiscal code validation, thank you!